src/EventStore.Core/ClusterVNode.cs

```diff
@@ -610,13 +607,15 @@ public class ClusterVNode<TStreamId> :
 					out streamLastEventNumberCache,
 					out streamMetadataCache,
 					out streamInfoCacheResizer);
-			else
+			else {
```

Comment:
better not to add in the curly braces here since there aren't any in the 'if'. it's a small thing but things like this add up to make the code easier/harder to read

Reply:
sure, will make the changes